contrib/cachetool.php

```diff
@@ -64,36 +66,51 @@
 set('cachetool_options', function () {
     $options = get('cachetool');
     $fullOptions = get('cachetool_args');
+    $return = [];
 
     if (strlen($fullOptions) > 0) {
-        $options = "{$fullOptions}";
+        $return = ["{$fullOptions}"];
```

Comment:
Is there a reason for "{ because imho you can remove it and just have `[$fullOptions]`

Reply:
It was there before. Did not change it. But yes, it is not needed.

Reply:
Thanks for the cleanup :-)